### ISSUE TITLE
Transfer downloaded shows to music files

### ISSUE THREAD

**Issue by DrGonzoinvt** [alecgorge/PhishOD-iOS#115]
It would be extra nice if after downloading a show one could save it a music program like iTunes or whatever to burn later. I don't know if there are legal issues with doing this or not. I would think not, but I've never been much for legal things anyway.
Just a suggestion.

**Comment by alecgorge** [comment]
I like this idea but there are some limitations with iTunes file transfers with apps. Every source can be downloaded as a zip of mp3s and maybe making that more apparent with a way to email it to yourself might be just as good?

**Comment by DrGonzoinvt** [comment]
Yes, almost just as good. But instructions would definitely be needed, especially for the gray matter challenged such as myself.
Thanks for the quick reply. The App is awesome.

John
